INTERVIEW_TYPE

```diff
@@ -0,0 +1,3 @@
+# Before starting interview, uncomment the desired line below:
```

Comment:
I think we need to add some details on this to the README, for example this has to be set before setting up and later on tearing down the interview infra.